src/runtime/virtcontainers/qemu.go

```diff
@@ -358,24 +377,29 @@ func (q *qemu) appendImage(ctx context.Context, devices []govmmQemu.Device) ([]g
 }
 
 func (q *qemu) createQmpSocket() ([]govmmQemu.QMPSocket, error) {
-	monitorSockPath, err := q.qmpSocketPath(q.id)
-	if err != nil {
-		return nil, err
-	}
+	qmpPaths, err := q.qmpSocketPaths(q.id)
+        if err != nil {
```

Comment:
Replace these spaces with tabs. The `go fmt` check should fail.

Reply:
done.

Reply:
done